packages/melody-compiler/__tests__/__snapshots__/CompilerSpec.js.snap

```diff
@@ -2235,6 +2228,80 @@ http://twig.sensiolabs.org/doc/filters/index.html"
 
 exports[`Compiler should fail transforming wrong import.template 1`] = `"Import must be used in macro, block or template"`;
 
+exports[`Compiler should transform templates with warnings mount without as.template 1`] = `
+"
+import Footwig from \\"foo.twig\\";
+import { component } from \\"melody-idom\\";
+import Component from \\"./component\\";
+export const _template = {};
+
+_template.render = function (_context) {
+  component(Component, \\"bg/=vtO\\");
+  component(Footwig, \\"%XboJwd\\", {
+    foo: \\"bar\\"
+  });
+  component(_context.Foo, \\"GwcMyf%\\");
+  component(_context.Foo, \\"$~m0VFY\\", {
+    foo: \\"bar\\"
+  });
+};
+
+if (process.env.NODE_ENV !== \\"production\\") {
+  _template.displayName = \\"MountWithoutAs\\";
+}
+
+export default function MountWithoutAs(props) {
+  return _template.render(props);
+}
+"
+`;
+
+exports[`Compiler should transform templates with warnings mount without as.template 2`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [
+      "Missing 'as' key with mount.
+> 1 | {% mount './component' %}
+    |    ^
+  2 | {% mount 'foo.twig' with { foo: 'bar' } %}
+  3 | {% mount Foo %}
+  4 | {% mount Foo with { foo: 'bar' } %}
+
+There should be a unique key passed with \\"as\\" keyword. e.g: {% mount './component' as 'comp' %}",
+    ],
+    Array [
+      "Missing 'as' key with mount.
+  1 | {% mount './component' %}
+> 2 | {% mount 'foo.twig' with { foo: 'bar' } %}
```

Comment:
If you pass in a different length parameter you should be able to create an underline for the offending `mount` statement. That should make it easier to understand.